src/WignerSymbols.jl

```diff
@@ -365,13 +448,83 @@ function compute6jseries(β₁, β₂, β₃, α₁, α₂, α₃, α₄)
     return Base.unsafe_rational(totalnum, totalden)
 end
 
+# compute the sum appearing in the 9j symbol
+function compute9jseries(a, b, c, d, e, f, g, h, j)
+    I1 = max(abs(h - d), abs(b - f), abs(a - j))
+    I2 = min(h + d, b + f, a + j)
+    krange = I1:I2
+ 
+    sum(krange) do k
+        p = iseven(2k) ? big(2k + 1) : -big(2k + 1)
+
+        b₁ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (a, b, c, f, j, k)     
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
+            wei9jbracket(α₁, α₂, α₃, β₁, β₂, β₃, β₀)
+        end
+        
+        b₂ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (f, d, e, h, b, k)
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
+            wei9jbracket(α₁, α₂, α₃, β₁, β₂, β₃, β₀)
+        end
+
+        b₃ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (h, j, g, a, d, k) 
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
```

Comment:
Was there a specific reason for calling the last one `β₀` and not `β₄`?

Reply:
Also, the construction of the `α` and `β` is the same for `b₁`, `b₂` and `b₃`. Was there a specific reason not to include this as the start of the `wei9jbracket` function, such that here you could simply have
```julia
b₁ = wei9jbracket(a, b, c, f, j, k)
b₂ = wei9jbracket(f, d, e, h, b, k)
b₃ = wei9jbracket(h, j, g, a, d, k)
```

Reply:
No real reason - loosely the last one stood out to me as a break in a pattern since it doesn't have a matching αᵢ (wrt. the `mᵢ` used to form it), so I picked the 0 subscript to distinguish it a bit at a glance. I'd be happy to relabel it to `β₄` if you prefer!

Reply:
wrt. the `b₁`, `b₂`, `b₃` I was trying to maintain readability against the [Wei paper](https://pubs.aip.org/aip/cip/article/12/6/632/510234/New-formula-for-9-j-symbols-and-their-direct) - but was also generally following the pattern from the 3j and 6j code where conversions tended to be performed before passing into a more specialized computational function. In this case, the `mᵢ` are generally `BigHalfInt`s but the relevant linear combinations always come out convertible to `BigInt`s, which are what the `primebinomial` calls in `wei9jbracket` were going to need. This could be refactored pretty easily if you prefer.